LocalSettings.php

```diff
@@ -4100,6 +4111,38 @@
 				'edit' => true,
 			),
 		),
+		'+wikicanadawiki' => array(
+			'bureaucrat' => array(
+				'managechangetags' => true,
+				'deletechangetags' => true,
+				'importupload' => true,
+				'import' => true,
+				'markbotedits' => true,
+				'mergehistory' => true,
+				'abusefilter-modify' => true,
+				'abusefilter-modify-restricted' => true,
+				'changetags' => true,
+				'applychangetags' => true,
+				'editcontentmodel' => true,
+			),
+			'sysadmin' => array(
```

Comment:
We've explained this several times. CheckUser and Oversight are "steward" permissions and not "sysadmin" permissions